pyproject.toml

```diff
@@ -42,4 +40,13 @@ build-backend = "hatchling.build"
 [tool.pytest.ini_options]
 addopts = [
     "--import-mode=importlib",
-]
\ No newline at end of file
+]
+norecursedirs = [
+    "tests/helpers",
+]
+
+[tool.uv]
```

Comment:
I think I'm good with adding `uv` as a tool dependency as part of implementing CI. It makes sense here.